### PR TITLE
CI: Azure CPU pool

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

switch agent pool

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @carmocca @akihironitta @borda

### CODE REVIEW

**Review by manskx — APPROVED** [review]

**Comment by Borda** [comment]
for visibility: https://github.com/microsoft/containerregistry/issues/123

File: .azure/app-cloud-e2e.yml [hunk 1]
```diff
@@ -80,10 +83,10 @@ jobs:
 
     - bash: |
         python -m pip install playwright
-        python -m playwright install --with-deps
+        python -m playwright install  # --with-deps
```
Comment (Borda) [inline .azure/app-cloud-e2e.yml:86]: Disable the dependencies as we use playwright image; maybe skip this step completely?

**Review by awaelchli — APPROVED** [review]

File: .azure/app-cloud-e2e.yml (hunk 1 above)
Comment (manskx) [reply .azure/app-cloud-e2e.yml:86]: yes, let's try to remove this step completely if we already use playwright image

Comment (Borda) [reply .azure/app-cloud-e2e.yml]: trying now... :pray:

**Review by manskx — APPROVED** [review]
🚀

**Review by akihironitta — APPROVED** [review]

**Comment by awaelchli** [comment]
The CI on forks still seems to fail for the same reason even after this PR:

https://dev.azure.com/Lightning-AI/lightning/_build/results?buildId=92773&view=logs&j=77edbfcc-d10a-5913-888c-53723d747136&t=05490af5-150e-5d97-94e5-647c4ee95338

```
requests.exceptions.MissingSchema: Invalid URL '$(LIGHTNING_CLOUD_URL_PROD)/v1/auth/login': No scheme supplied. Perhaps you meant [http://$(LIGHTNING_CLOUD_URL_PROD)/v1/auth/login?](http://%24%28lightning_cloud_url_prod%29/v1/auth/login?)
```

Where can we fix this URL to include https?

**Pull request merged**